cspell.json

```diff
@@ -0,0 +1,45 @@
+{
+    "version": "0.1",
+    "enableFiletypes": [
+        "git-rebase",
+        "ini",
+        "pip-requirements"
+    ],
+    "flagWords": [
+        "colour",
+        "hte",
+        "optimise"
+    ],
+    "ignorePaths": [
+        "cspell.json"
+    ],
+    "language": "en-US",
+    "words": [
+        "docstrings",
+        "expertsystem",
+        "jupyter",
+        "mypy",
+        "numpy",
+        "permalinks",
+        "pydocstyle",
+        "pylint",
+        "pytest",
+        "repos",
+        "rstcheck",
+        "xmltodict"
+    ],
+    "ignoreWords": [
+        "commit's",
+        "envlist",
+        "esbenp",
+        "htmlcov",
+        "ipynb",
+        "nbsphinx",
+        "nbstripout",
+        "oneline",
+        "pylintrc",
+        "pyproject",
+        "rcfile",
+        "rtfd"
```

Comment:
how do you differentiate between the two categories? cspell says words would appear as suggestions, not really sure this works with vscode nicely

Reply:
https://github.com/ComPWA/expertsystem/pull/90/files#diff-6a35e6d99715b7e5c07424b1f7f9fea0R71-R75

Reply:
I tested it in vscode, the suggestions work well!